tests/tests_fabric/helpers/runif.py

```diff
@@ -141,6 +142,16 @@ def __new__(
         )
 
 
+def skip_if_dynamo_unsupported():
```

Comment:
lets add it to `RunIf`

Reply:
How should I do it? You mean `RunIf(dynamo=True|False)`. Is it worth it?

Reply:
I opened https://github.com/Lightning-AI/lightning/pull/17404